backend/src/panels/AbstractWebviewPanel.ts

```diff
@@ -20,6 +20,7 @@ export abstract class AbstractWebviewPanel {
 
 	protected logger: IChildLogger;
 	protected disposables: vscode.Disposable[];
+	protected inTheia: boolean;
```

Comment:
protected isInTheia: boolean;

Reply:
There is already method with this name

Reply:
youhave set it onAbstractWebviewPanel but didn't use it.

Reply:
It is declared once in this abstract panel but used in both other panels: yeomanui panel and explogen panel

Reply:
I see now. olease use same param names in both panel --> isInTheia: boolean